editPage.html

```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
```

Comment:
I love what you did with the mobile view but the desktop is not the same as the design, kindly work on this.

This is the original design https://www.figma.com/file/yD5F6g67uY8Gj6GytXXj9S/Team-124-cc_gen-Wireframes?node-id=1190%3A52430.

And this what we have now 
![image](https://user-images.githubusercontent.com/60205816/183938395-44058c1e-761f-420b-b5a6-8022b48d34a0.png)